src/main/kotlin/ai/digital/integration/server/release/util/ReleaseServerUtil.kt

```diff
@@ -153,9 +153,21 @@ class ReleaseServerUtil {
         }
 
         fun runDockerBasedInstance(project: Project) {
-            project.exec {
-                executable = "docker-compose"
-                args = listOf("-f", getResolvedDockerFile(project).toFile().toString(), "up", "-d")
+            // Use ProcessBuilder instead of deprecated execOperations
+            val command = listOf("docker-compose", "-f", getResolvedDockerFile(project).toFile().toString(), "up", "-d")
+            val processBuilder = ProcessBuilder(command)
```

Comment:
Why isn't the **execOperations.exec** method being used in this case?

Reply:
Updated with execOperations.exec method